gitlab.go

```diff
@@ -187,9 +187,6 @@ func (c *Client) SetBaseURL(urlStr string) error {
 		return err
 	}
 
-	// Set the encoded opaque data
-	c.baseURL.Opaque = c.baseURL.Path
-
```

Comment:
Very nice, wasn't aware of that change/commit :smile: I'm fine with moving to Go 1.5, but we should mention this minimum requirement very clearly in the README.